x-pack/solutions/observability/plugins/slo/server/services/transform_generators/common.test.ts

```diff
@@ -77,5 +79,28 @@ describe('common', () => {
         },
       });
     });
+
+    it('starts at now minus 7 days when preventInitialBackfill is false and serverless is true', () => {
+      expect(
+        getFilterRange(
+          createSLO({
+            timeWindow: sevenDaysRolling(),
```

Comment:
I would use a thirtyDaysRolling() so we ensure we are not using the sevendDays as the range filter on serverless